src/material-examples/cdk-datepicker-overview/cdk-datepicker-overview-example.html

```diff
@@ -0,0 +1,10 @@
+<input [cdkDatepicker]="picker">
+<cdk-datepicker #picker>
+  <my-calendar (selectedChange)="_dateSelected()"></my-calendar>
```

Comment:
should this `(selectedChange)` go on the `cdk-datepicker` instead?

Reply:
I don't think so. The `my-calendar` extends from `CalendarView`, which has the `selectedChange` property. The `cdk-datepicker` doesn't have the same property.